README.md

```diff
@@ -2,13 +2,14 @@
 
```

Comment:
Suggest changing the CI badge, since this one points to AB3 instead of ours. Will supply the correct link later.